views/pages/funding/under10k.njk

```diff
@@ -19,7 +19,7 @@
         )
     }}
 
-    <article role="main" class="nudge-up">
+    <main class="nudge-up">
```

Comment:
I think the `role` here does some useful a11y stuff, or does using `<main>` do the same thing?

Reply:
`<main>` does the same thing afaik. It has the implicit role of `main` whereas `<article role="main">` is being assigned a role of `main`…if that makes sense.